src/python/Utils/TokenManager.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python
+#-*- coding: utf-8 -*-
+#pylint: disable=W0212
+"""
+File       : TokenManager.py
+Author     : Valentin Kuznetsov <vkuznet AT gmail dot com>
+Description: IAM token manager
+"""
+
+# system modules
+import os
+import ssl
+import time
+import logging
+import traceback
+
+# third part library
+try:
+    import jwt
+except ImportError:
+    traceback.print_exc()
+    jwt = None
+
+# prevent "SSL: CERTIFICATE_VERIFY_FAILED" error
+# this will cause pylint warning W0212, therefore we ignore it above
+ssl._create_default_https_context = ssl._create_unverified_context
+
+
+def getToken(name=None):
+    """
+    Get IAM token either from environment or file name
+    :param name: ether file name containing token or environment name which hold the token value.
+    If not provided it will be assumed to read token from IAM_TOKEN environment.
+    :return: token or None
+    """
+    if name and os.path.exists(name):
+        token = None
+        with open(name) as istream:
+            token = istream.read()
+        return token
+    if name:
+        return os.environ.get(name)
+    return os.environ.get("IAM_TOKEN")
```

Comment:
I'd suggest to actually suggest a global constant with this default environment variable name, e.g. (at the top of the module):
```
DEFAULT_TOKEN_ENV="IAM_TOKEN"
```
what do you think?

Reply:
to Alan: why do you want to invent and introduce yet another global variable, I always against all globals.
to Todor: why do you want to complicate handling of different use-case by adding more code? In current implementation you can either read token from env or from file using the same function. In your proposal, not only we need two separate functions here, but also on a client side you'll need to call them independently. Why do you want to duplicate behavior while gaining nothing?

Reply:
Thanks Valentin, I'm resolving this as well.